src/hotspot/cpu/aarch64/aarch64_sve.ad

```diff
@@ -250,7 +249,7 @@ definitions %{
 
 // All SVE instructions
 
-// vector load/store
+// vector load/store/reinterpret
```

Comment:
Can you please add a separate comment for "reinterpret" before the related rules?

Reply:
This comment is copied from neon implementation. We will add a  separate comment for "reinterpret"  in revised patch.